src/main/java/org/elasticsearch/cluster/block/ClusterBlock.java

```diff
@@ -131,7 +131,7 @@ public void readFrom(StreamInput in) throws IOException {
         final int len = in.readVInt();
         ArrayList<ClusterBlockLevel> levels = new ArrayList<>();
         for (int i = 0; i < len; i++) {
```

Comment:
Given that we changed the serialization code for this class, would it be possible to write a small unit test that verifies it?  We already have similar `*RequestTests#testSerialization` tests that randomize the version and make sure that everything is fine.